pkgs/development/libraries/stduuid/default.nix

```
@@ -1,4 +1,4 @@
-{ stdenv, cmake, fetchFromGitHub, lib }: let
+{ stdenv, cmake, fetchFromGitHub, fetchpatch, lib }: let
   version = "1.2.3";
 in stdenv.mkDerivation {
   name = "stduuid-${version}";
```

Well, this expression is certainly very old.

```suggestion
  pname = "stduuid";
```

---

Sure, I should also add `inherit version;` right?

---

Done